### ISSUE TITLE
"Form based tag editor" mystery 

### ISSUE THREAD

**Issue by jidanni** [MarcusWolschon/osmeditor4android#1993]
In https://vespucci.io/help/en/Advanced%20preferences/ we see exactly two mentions of a "form based tag editor".

Well I changed the default from 4  but I am still not sure exactly what it affected. 

I'm thinking that maybe all this has something to do with the Properties panel.

Or maybe it has to do something with the Details Panel.

Anyway, in the documentation it it would be great if there was an example of how exactly to see what this setting/s affects.

Okay, I tried toggling it off, and the Details Panel became the Properties Panel so there was one less panel.

In other words what looks like the Details Panel is now named the Properties Panel. And what looked like the Properties Panel, is now gone.

 Okay now I know what this is all about. So the documentation perhaps could use some of this more modern terminology.

**Comment by jidanni** [comment]
Okay I'm changing the default of four to a small number gives:
![Screenshot_20221215-174734_Vespucci.jpg](https://user-images.githubusercontent.com/1230959/208003670-988eacc0-7b54-499b-9a0d-dca8f27a4862.jpg)![Screenshot_20221215-175008_Vespucci.jpg](https://user-images.githubusercontent.com/1230959/208003699-ccfe6b84-c501-4eae-b184-05107773a993.jpg)
And to a large number, gives:
![Screenshot_20221215-174554_Vespucci.jpg](https://user-images.githubusercontent.com/1230959/208003749-5d827940-c850-442f-b982-f4666bde7f16.jpg)
So the effects seem sort of multiple.

**Comment by simonpoole** [comment]
There is no mystery, the option controls exactly what the description in the documentation says:
"Maximum number of values that will directly be displayed in the form based editor for Tags with pre-determined values. "

**Comment by jidanni** [comment]
These two preferences should be grouped: Toggling it on should reveal   the other, where the user can pick the number in a sub menu.
